libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef-bl.dat

```diff
@@ -41,6 +41,8 @@ PA13 JTMS-SWDIO SWD
 PA14 JTCK-SWCLK SWD
 
 PA2 LED_BOOTLOADER OUTPUT LOW
+PD15 OUTPUT HIGH PULLUP
```

Comment:
this doesn't look right, it is missing a label. Maybe you mean this:
PD15 BUZZER OUTPUT HIGH